v2/src/log_context_property_basic_types.c

```diff
@@ -13,149 +14,209 @@
-            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_027: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int32_t).copy shall copy the bytes of the int32_t value from the address pointed by src_value to dst_value. ]*/ \
-            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_037: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint32_t).copy shall copy the bytes of the uint32_t value from the address pointed by src_value to dst_value. ]*/ \
-            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_047: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int16_t).copy shall copy the bytes of the int16_t value from the address pointed by src_value to dst_value. ]*/ \
-            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_057: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint16_t).copy shall copy the bytes of the uint16_t value from the address pointed by src_value to dst_value. ]*/ \
-            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_067: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int8_t).copy shall copy the byte of the int8_t value from the address pointed by src_value to dst_value. ]*/ \
-            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_077: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint8_t).copy shall copy the byte of the uint8_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_007: [ LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(int64_t).copy shall copy the bytes of the int64_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_017: [ LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(uint64_t).copy shall copy the bytes of the uint64_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_027: [ LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(int32_t).copy shall copy the bytes of the int32_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_037: [ LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(uint32_t).copy shall copy the bytes of the uint32_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_047: [ LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(int16_t).copy shall copy the bytes of the int16_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_057: [ LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(uint16_t).copy shall copy the bytes of the uint16_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_067: [ LOG_CONTEXT_PROPERTY_TYPE_IF_IMPL(int8_t).copy shall copy the byte of the int8_t value from the address pointed by src_value to dst_value. ]*/ \
```

Comment:
>int8_t [](http://example.com/codeflow?start=102&length=6)

We should strive not to do a memcpy for 1 byte of "int8_t...

Reply:
Changed to assigning for int8_t/uint8_t